NAMESPACE

```diff
@@ -30,5 +30,6 @@ import(data.table)
 import(methods)
 import(nc)
 import(qs)
+import(utils)
 import(xml2)
 importFrom(utils,unzip)
```

Comment:
typically to avoid confusion, you should not use both importFrom(utils,unzip) and import(utils). what other function from utils are you using? Please either add that to the importFrom, or remove the importFrom, keep the import, and add utils:: prefix to the R code from utils, for example utils::unzip

Reply:
I'm using `unzip` and `install.packages` from utils. If I use `import` for utils, I get the following warning: 
```
Warning: replacing previous import ‘Rcpp::prompt’ by ‘utils::prompt’ when loading ‘pkgA’
```
There is most likely some conflict between utils and Rcpp. Instead, I replaced `import` with `importFrom` with commit 11eaa1b.